samples/react-sample-app/public/runtime-config.json

```diff
@@ -1,9 +1,9 @@
 {
-  "clientID": "Tus9QHgOFnb17V0SCayvy0u8vmsa",
-  "baseUrl": "https://localhost:9443",
+  "clientID": "XiHfEM3FNS88gmVM2JwH7XBYrKga",
```

Comment:
Remove these changes

Reply:
Done in [7971638](https://github.com/AfraHussaindeen/identity-verification-onfido/pull/3/commits/7971638e82c06f3727cdc16fe0ab0e6005e4232e)[7971638](https://github.com/AfraHussaindeen/identity-verification-onfido/pull/3/commits/7971638e82c06f3727cdc16fe0ab0e6005e4232e)